views/trainers/new_trainer.erb

```diff
@@ -0,0 +1,13 @@
+<h2>To Create A New Trainer, Please Fill Out The Below Form</h2>
+<form action="/trainer" method="post">
+  <label>Name: </label>
+  <input type:"text" name="trainer[name]">
+
+  <label>Level: </label>
+  <input type:"text" name="trainer[power_level]">
+
+  <label>Image Url: </label>
+  <input type:"text" name="trainer[img_url]">
+
+  <input type="submit" value="CREATE">
```

Comment:
don't think you need CREATE here-- the action (verb) and method params (path) on line 2 will create a request get caught by a `post "/trainer"` route